tests/integrations/notdiamond/cassettes/tracing_test/test_notdiamond_quickstart.yaml

```diff
@@ -0,0 +1,1677 @@
+interactions:
+- request:
+    body: '{"query": "query DefaultEntity {\n  viewer {\n    username\n    defaultEntity
+      {\n      name\n    }\n  }\n}"}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate, zstd
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '108'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.32.3
+    method: POST
+    uri: https://api.wandb.ai/graphql
```

Comment:
calls to api.wandb.ai should never be included in cassettes. This indicates the test was recorded outside of the standard client fixture